assignment/week3/lunchMenu/src/pages/OnBoardingPage.jsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import Header from '../components/Header';
+import SelectCategory from '../components/SelectCategory';
+import RecommendStartPage from './RecommendStartPage';
+const OnBoardingPage = () => {
+  const [category, setCategory] = useState('');
+  const [startRecommend, setStartRecommend] = useState(false);
+
+  return (
+    <>
+      {category ? (
+        <>
+          <Header category={category} setCategory={setCategory} />
+          <Style.Container>
+            <RecommendStartPage
+              category={category}
+              startRecommend={startRecommend}
+              setStartRecommend={setStartRecommend}
+            />
+          </Style.Container>
+        </>
+      ) : (
+        <>
+          <Header />
+          <Style.Container>
+            <SelectCategory setCategory={setCategory} />
+          </Style.Container>
+        </>
+      )}
+    </>
```

Comment:
엇 `<Header>`랑 `<Style.Container>`는 중복되는 컴포넌트이니, 삼항연산자는 `Style.Container` 안에서 써줘도 되겠어요!

```suggestion
    <>
      <Header category={category} setCategory={setCategory} />
      <Style.Container>
        {category ? (
          <RecommendStartPage
            category={category}
            startRecommend={startRecommend}
            setStartRecommend={setStartRecommend}
          />
        ) : (
          <SelectCategory setCategory={setCategory} />
        )}
      </Style.Container>
    </>
```

이렇게~~

Reply:
오오오 좋아용 !!!